src/tests/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 Viam Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+enable_testing()
+include(viamcppsdk_utils)
+viamcppsdk_add_boost_test(test_generic_component.cpp)
+
+add_executable(viam_cpp_sdk_tests
```

Comment:
Do you still need this line? It looks to me like `viamcppsdk_add_boost_test` will call `add_executable` and `target_link_libararies` for you.

Reply:
quite right! fixed